### PR TITLE
[lab-intro-prob] Filipe Koch

### PR DESCRIPTION
**Pull request by fckoch**

### CODE REVIEW

**Comment by ta-data-lis** [comment]
Lab checked, good job!

**Challenge 4**
If you define a 2 entries table with the gender on the rows and the answers in the index you would find yourself with something like this:

              | YES | NO | TOT
     MEN | 324 | 156 | 480
WOMEN| 351 | 169 | 520

You want to check if the probability of willing or not more lights (YES/NO) is actually depending on 
the gender.
So, you just have to calculate the 2 according to our observations:
P(YES I MEN) = 324/480 = 0.675
P(YES) = (324+351)/(480+520) = 0.675

Since the probability of willing more lights conditioned to the fact of the respondent being a man is the same as the probability of willing more lights regardless of the gender then the probability of willing more lights is independent from the probability of the respondent being a man.

In other words, the gender is not influencing the response to the lights survey.


The rest of the answers are all correct! Please, next time just explicit your process a little more.

Keep up the good work!
